src/lib/DpApi.js

```diff
@@ -123,8 +129,8 @@ const handleResponseMessages = function (responseMeta) {
  * Perform rudimentary response validation, handle response messages.
  *
  * @param {Object} response
- *                    The axios wrapper around the XMLHttpRequest instance.
- *                    See https://github.com/axios/axios#response-schema for documentation.
+ *                    The response provided by the fetch API around the XMLHttpRequest instance.
```

Comment:
Should the "around the XMLHttpRequest instance." part still be here? I don't understand the meaning :thinking:

Reply:
cood catch b6fc5709ec131e3a6e3cad7fac3c1a9ebf23664d